pdcupdater/utils.py

```diff
@@ -527,7 +527,38 @@ def pkgdb2release(collection):
     return "-".join(release)
 
 
-def tag2release(tag):
+def tag2release(tag, pdc=None):
+    if pdc:
+        return _tag2release_with_pdc(pdc, tag)
+    else:
+        return _tag2release_fedora(tag)
+
+
+def _tag2release_with_pdc(pdc, tag):
+    """ Use PDC to figure out what release is associated with this tag. """
+    releases = list(pdc.get_paged(
+        pdc['releases']._,
+        brew_allowed_tag=tag,
```

Comment:
I heard back that there's a "brew" extension to PDC internally, but there's nothing private about it other than the naming of "brew" vs "koji".